### PR TITLE
Fix for a partially broken columns Fix #173

### LINKED ISSUE #173: Row partially broken for SUCCESS jobs
I've noticed with the addition of the "Failed Tempest Tests" column reports are being generated incorrectly for SUCCESS jobs for that column and the Blockers column

Examples:
- http://psi-jeeves.css-qe.com/archive/enterprise_report_2020-12-29_12-01-55.html
- http://psi-jeeves.css-qe.com/archive/storage_report_2020-12-29_09-07-18.html

@itzikb Could you please take a look at this?

### PR DESCRIPTION
**Pull request by itzikb-redhat**
Fixing a problem where the columns order is wrong when a job is successful

### CODE REVIEW

**Review by nathan-weinberg — COMMENTED** [review]
LGTM, @agarcia64 can you test this with Storage DFG report?

File: templates/report_template.html [hunk 1]
```diff
@@ -69,13 +69,12 @@
 							<td style="text-align: center;">{{row.build_days_ago}}</td>
 							<td style="text-align: center;">{{row.compose}}</td>
 							<td style="text-align: center;" bgcolor="#3465a4"><a href="https://www.youtube.com/watch?v=68ugkg9RePc" style="text-decoration: none; color: #000;">{{row.lcb_result}}</a></td>
-							<td style="text-align: center;"><p>N/A</p></td>
 							{% if row.tempest_tests_failed != None %}
 								<td style="text-align: center;"><a href="{{row.tempest_tests_url}}">{{row.tempest_tests_failed}}</a></td>
 							{% else %}
 								<td style="text-align: center;">N/A</td>
 							{% endif %}
```
Comment (nathan-weinberg) [inline templates/report_template.html:76]: Do we need this check? When will a successful job have failed tempest tests?

Comment (itzikb) [reply templates/report_template.html:76]: It was a precaution. I'll remove

**Review by agarcia64 — APPROVED** [review]
Tested the latest commit on my machine, everything is working as expected and the code looks good!

File: templates/report_template.html [hunk 2]
```diff
@@ -69,13 +69,8 @@
 							<td style="text-align: center;">{{row.build_days_ago}}</td>
 							<td style="text-align: center;">{{row.compose}}</td>
 							<td style="text-align: center;" bgcolor="#3465a4"><a href="https://www.youtube.com/watch?v=68ugkg9RePc" style="text-decoration: none; color: #000;">{{row.lcb_result}}</a></td>
+							<td style="text-align: center;">N/A</td>
```
Comment (nathan-weinberg) [inline templates/report_template.html:72]: This should be nested in a <p> tag, same as the line below

**Review by nathan-weinberg — APPROVED** [review]

**Pull request merged**